x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_overview.tsx

```diff
@@ -188,13 +188,15 @@ export const AgentDetailsOverviewSection: React.FunctionComponent<{
             description: (agent.tags ?? []).length > 0 ? <Tags tags={agent.tags ?? []} /> : '-',
           },
         ].map(({ title, description }) => {
+          const tooltip =
+            typeof description === 'string' && description.length > 20 ? description : '';
           return (
             <EuiFlexGroup>
               <FlexItemWithMinWidth grow={3}>
                 <EuiDescriptionListTitle>{title}</EuiDescriptionListTitle>
               </FlexItemWithMinWidth>
               <FlexItemWithMinWidth grow={7}>
-                <EuiDescriptionListDescription className="eui-textTruncate">
+                <EuiDescriptionListDescription className="eui-textTruncate" title={tooltip}>
```

Comment:
Is it possible for this to be an `EuiTooltip` instead? I think we should only use the `title` attribute if absolutely necessary, preferring our design system tooltips.

Reply:
updated